desec.py

```diff
@@ -150,7 +150,7 @@ def __init__(self, token, retry_limit=3):
         self._token_auth = TokenAuth(token)
         self._retry_limit = retry_limit
 
-    def query(self, method, url, data=None):
+    def query(self, method, url, data=None, jsonEncode=True):
```

Comment:
Please remove the parameter. Its not needed. Use the `Content-Type` response header to handle the response correctly.

Reply:
Removed jsonEncode and added Content-Type Header parsing.